composer.json

```diff
@@ -29,9 +29,15 @@
     "require-dev": {
         "phpstan/phpstan": "^1.0",
         "phpunit/phpunit": "^9.5",
-        "squizlabs/php_codesniffer": "^3.7"
+        "squizlabs/php_codesniffer": "^3.7",
+        "friendsofphp/php-cs-fixer": "^3.11"
     },
     "scripts": {
         "lint": "phpstan && phpcs ."
+    },
+    "config": {
+        "platform": {
+            "php": "8.0.13"
```

Comment:
I've got PHP 8.1 on my host so it'll default to installing dependencies targeting this version.